BuildTransportPackage.ps1

```diff
@@ -66,15 +68,11 @@ if(-not (test-path "$fullNugetPath\build\native"))
     new-item -path "$fullNugetPath\build\native" -itemtype "directory"
 }
 
-if(-not (test-path "$fullNugetPath\mrt_raw"))
+if(-not (test-path "$fullNugetPath\lib\net5.0-windows10.0.17763.0"))
```

Comment:
Nice but what's it got to do with EM?

Reply:
This wasn't supposed to be in here.  I'll remove it.